build.gradle

```diff
@@ -31,6 +31,9 @@ allprojects {
     targetCompatibility = '1.8'
 
     repositories {
+        maven {
```

Comment:
This didn't seem to work, it complained about not being able to find the mesos 1.2 snapshot when I moved it into the scheduler's `build.gradle`.